### PR TITLE
MMT-1157 implement datetime pickers for bulk update searching

### PR DESCRIPTION
**Pull request by macrouch**

### CODE REVIEW

File: spec/features/bulk_updates/bulk_updates_search_spec.rb [hunk 1]
```diff
@@ -73,5 +73,40 @@
...
+        within '#collection-search-results' do
+          expect(page).to have_css('tbody > tr', count: 5)
+        end
+      end
+    end
+
+    context 'when searching for collections by Temporal' do
+      before do
+        within '#collection-search' do
+          select 'Revision Date', from: 'Search Field'
```
Comment (charleshuang80) [inline spec/features/bulk_updates/bulk_updates_search_spec.rb]: The context says searching by 'Temporal' but this is selecting `Revision Date`

File: app/assets/javascripts/datetimepicker.coffee [hunk 2]
```diff
@@ -8,7 +8,7 @@ $(document).ready ->
     format: 'YYYY-MM-DD[T]HH:mm:ss[Z]'
 
     # Sets the minimum date to the start of today
-    minDate: moment().startOf('day')
+    # minDate: moment().startOf('day')
```
Comment (charleshuang80) [inline app/assets/javascripts/datetimepicker.coffee]: if this isn't being used, can these comments be removed?

File: spec/features/bulk_updates/bulk_updates_search_spec.rb [hunk 3]
```diff
@@ -22,7 +22,7 @@
 
     it 'displays the correct options for the search field' do
       within '#collection-search' do
-        options = BulkUpdatesHelper::SEARCHABLE_KEYS.map { |key, value| value[:title] }
+        options = BulkUpdatesHelper::SEARCHABLE_KEYS.map { |_key, value| value[:title] }
 
         expect(page).to have_select('field', options: options)
       end
```
Comment (charleshuang80) [inline spec/features/bulk_updates/bulk_updates_search_spec.rb]: comment for line 31 (but the closest github is letting me get is 28):
Should have thought of this earlier (sorry) - should we have a test like this for the `require_from_group` validation rule you added for the `double_date` fields?

Comment (macrouch) [reply spec/features/bulk_updates/bulk_updates_search_spec.rb]: Probably a good idea

**Pull request merged**